home/templates/home/tags/meta_tags.html

```diff
@@ -0,0 +1,19 @@
+{% load wagtailimages_tags %}
+
+{% with site_name=page.get_site.site_name %}
+    <meta property="og:url" content="{{ page.url }}">
+    <meta property="og:site_name" content="{{ site_name }}">
+    <meta property="og:title" content="{% firstof page.seo_title page.title %}">
+    <meta property="og:description" content="{% firstof page.search_description site_name %}">
+    <meta property="og:type" content="article">
+    <meta property="twitter:site" content="{{ site_name }}">
+    <meta property="twitter:title" content="{% firstof page.seo_title page.title %}">
+    <meta property="twitter:description" content="{% firstof page.search_description site_name %}">
+    <meta property="twitter:card" content="summary_large_image">
+    {% if page.lead_image %}
+        {% image page.lead_image width-360 as lead_image %}
```

Comment:
This has been hard-coded to 'width-360' and I was expecting this to be based on `IMAGE_SIZE_PRESET` (full or half-width). I think it would be less surprising to use the same image sizes throughout, and for sites that use larger images there won't be a need to create extra image renditions just for social media previews.